### PR TITLE
Add dumb bean tag

### PR DESCRIPTION
**Pull request by darkerbit**

### CODE REVIEW

**Review by gdude2002 — CHANGES_REQUESTED** [review]
You're missing a newline at the end of the file!

File: tags/fun/bean.ytag [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+embed:
+  title: User beaned
+  footer:
+    text: "ID: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx • xx/xx/xxxx"
+
+---
+{{0}} has been beaned until Dec 24, 2063 at 07:23 (UTC).
+
+Infraction Reason: {{1}}
```
Comment (gdude2002) [inline tags/fun/bean.ytag:13]: `Infraction Reason:` should be bold.

**Comment by darkerbit** [comment]
Done!

File: tags/fun/bean.ytag [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+embed:
+  title: User beaned
+  footer:
+    text: "ID: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx • xx/xx/xxxx"
+
+---
+
+{{0}} has been beaned until Dec 24, 2063 at 07:23 (UTC).
+
+*Infraction Reason:* {{1}}
```
Comment (Scotsguy) [inline tags/fun/bean.ytag:14]: Nope that's italic

Comment (darkerbit) [reply tags/fun/bean.ytag]: Fixed, you may have noticed I am not the smartest of individuals

**Review by gdude2002 — APPROVED** [review]

File: tags/fun/bean.ytag [hunk 3]
```diff
@@ -0,0 +1,14 @@
+type: embed
+
+colour: #2ECC71
```
Comment (gdude2002) [inline tags/fun/bean.ytag:3]: That's a comment, surround it in `"`

Comment (darkerbit) [reply tags/fun/bean.ytag]: welp

**Pull request merged**